src/index.ts

```diff
@@ -147,6 +147,8 @@ export type {
   GraphQLAbstractType,
   GraphQLWrappingType,
   GraphQLNullableType,
+  GraphQLNullableInputType,
+  GraphQLNullableOutputType,
```

Comment:
Adds a new type so it's a "feature".
Please change the label.

Anything that adds new public import is a "feature".
and require new minor version, see: https://semver.org/#spec-item-7

Please change the label.